auslib/admin/views/forms.py

```diff
@@ -265,3 +295,41 @@ class ReadOnlyForm(Form):
     product = StringField('Product', validators=[InputRequired()])
     read_only = BooleanField('read_only')
     data_version = IntegerField('data_version', widget=HiddenInput())
+
+
+class ScheduledChangeNewReleaseForm(ScheduledChangeTimeForm):
+    name = StringField('Name', validators=[InputRequired()])
+    product = StringField('Product', validators=[InputRequired()])
+    data = JSONStringField({}, 'Data', validators=[InputRequired()], widget=FileInput())
+    data_version = IntegerField('data_version', widget=HiddenInput())
```

Comment:
Good catch, we don't. This probably ends up never being used because it's optional.